File: guanabara-java/src/com/danillo/algoritmos/lista02/Exercicio21.java
    package com.danillo.algoritmos.lista02;

import java.util.Scanner;

public class Exercicio21 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira um ano: ");
        int ano = entrada.nextInt();

        if((ano % 4 == 0 && ano % 100 != 0) || ano % 400 == 0){
            System.out.println("O ano " +ano+ " é bissexto.");
        } else {
            System.out.println("O ano " +ano+ " não é bissexto.");
        }
    }
}

File: guanabara-c/lista02/exercicio21.c
#include <stdio.h>

int main(){

    int ano;

    printf("Insira um ano: ");
    scanf("%d", &ano);

    if(ano % 4 == 0 && ano % 100 != 0 || ano % 400 == 0){
        printf("O ano %d é bissexto.", ano);
    }
    else{
        printf("O ano %d não é bissexto.", ano);
    }

    return 0;
}
